environments/python3/server.py

```diff
@@ -18,6 +17,19 @@ def load():
     userfunc = (imp.load_source('user', codepath)).main
```

Comment:
So what is `codepath` if the v1 endpoint is hit?  Won't this error out? Let's keep the v1 endpoints fully working for now-- we'll remove them in a few releases once v2 is stable.